max_xy_distance: 2.0
max_goal_iter: 100
odom_frame: "odom"
odom_topic: "/odometry/filtered"
cmd_topic: "/cmd_vel"
joint_states_topic: "/joint_states"
wheel_joint_name: 
- "front_left_wheel"
- "front_right_wheel"
steering_joint_name: 
- "front_left_steering_joint"
- "front_right_steering_joint"
distance_threshold: 0.15
goal_marker_topic: "/rl_controller/goal_marker"
is_rl_ackermann: true